File: eCommerce/ClientApp/src/Data/StaffPermission.ts
import {StorePermission} from "./StorePermission";

export class StaffPermission {
    public userId: string;
    public permissions: StorePermission[];
    
    constructor(userId: string, permissions: StorePermission[]) {
        this.userId = userId;
        this.permissions = permissions;
    }
}

File: eCommerce/Service/StaffPermission.cs
using System.Collections.Generic;
using eCommerce.Business;

namespace eCommerce.Service
{
    public class StaffPermission
    {
        public string UserId { get; }
        public IEnumerable<StorePermission> Permissions { get; }

        public StaffPermission(string userId, IEnumerable<StorePermission> permissions)
        {
            UserId = userId;
            Permissions = permissions;
        }
    }
}